app/src/main/java/app/revanced/integrations/utils/ReVancedUtils.kt

```diff
@@ -0,0 +1,46 @@
+package app.revanced.integrations.utils
+
+import android.content.Context
+import android.content.res.Resources
+import android.os.Handler
+import java.lang.ref.WeakReference
+
+private val CTX_NULL = IllegalArgumentException("Context cannot be null")
+private val CTX_NOT_INIT = IllegalArgumentException("Context is not initialized")
+
+object ReVancedUtils {
+    @JvmStatic
+    @get:JvmName("hasNewVideoStarted")
+    var hasNewVideoStarted = false
+
+    private var context: WeakReference<Context>? = null
```

Comment:
The motivation to use a `WeakReference` here was to not leak the `Context` since that is generally discouraged.